test/e2eTests/hooks.js

```diff
@@ -0,0 +1,11 @@
+const event = require('codeceptjs').event;
+const Apis = require('./rest/openMrsApis')
+
+module.exports = function() {
+	// Fired after every test
+  event.dispatcher.on(event.test.after, function (test) {
+  	// Remove any and all data that was injected into the DB
+  	// after each test
+  	Apis.cleanUp()
```

Comment:
@ynurmahomed @edrisse this line removes all data that was injected during a test. It allows tests to inject data without worrying about cleanup.

Reply:
@drryanjames ok, thx for explaining that. @ynurmahomed from my side i think we can merge.